src/routes/api/graphql.js

```diff
@@ -1,5 +1,15 @@
 import { createServer } from '@graphql-yoga/common';
 
+const sdls = Object.entries(import.meta.globEager('/src/lib/**/*.sdl.js')).map(([_, sdl]) => {
```

Comment:
maybe not needed due to [file-loading](https://www.graphql-tools.com/docs/schema-merging#file-loading) possibility

Reply:
Its needed, the mentioned package uses `require`, which isn't working with Sveltekit